core/block/import/importer.go

```diff
@@ -136,17 +141,23 @@ func (i *Import) Import(ctx context.Context,
 		i.s.ProcessAdd(progress)
 	}
 	i.recordEvent(&metrics.ImportStartedEvent{ID: importId, ImportType: req.Type.String()})
-	var rootCollectionId string
+	var (
+		rootCollectionId string
+		objectsCount     int64
+	)
+	returnedErr = fmt.Errorf("unknown import type %s", req.Type)
 	if c, ok := i.converters[req.Type.String()]; ok {
-		rootCollectionId, returnedErr = i.importFromBuiltinConverter(ctx, req, c, progress, origin)
-		return rootCollectionId, progress.Id(), returnedErr
+		rootCollectionId, objectsCount, returnedErr = i.importFromBuiltinConverter(ctx, req, c, progress, origin)
```

Comment:
Maybe we can return `ImportResponse` from `importFromBuiltinConverter` and than enrich it with progress.ID ?

Reply:
I don't think this is a good idea, because it's not obvious mutate of ImportResponse structure

Reply:
Yeah, agree